tsdef/types.d.ts

```diff
@@ -0,0 +1,156 @@
+
+import { LocalAudioTrack } from './LocalAudioTrack';
+import { LocalAudioTrackPublication } from './LocalAudioTrackPublication';
+import { LocalDataTrack } from './LocalDataTrack';
+import { LocalDataTrackPublication } from './LocalDataTrackPublication';
+import { LocalVideoTrack } from './LocalVideoTrack';
+import { LocalVideoTrackPublication } from './LocalVideoTrackPublication';
+import { RemoteAudioTrack } from './RemoteAudioTrack';
+import { RemoteAudioTrackPublication } from './RemoteAudioTrackPublication';
+import { RemoteDataTrack } from './RemoteDataTrack';
+import { RemoteVideoTrack } from './RemoteVideoTrack';
+import { RemoteVideoTrackPublication } from './RemoteVideoTrackPublication';
+import { Track } from './Track';
+import { VideoTrack } from './VideoTrack';
+
+export type LocalTrack = LocalAudioTrack | LocalVideoTrack | LocalDataTrack;
+export type RemoteTrack = RemoteAudioTrack | RemoteVideoTrack | RemoteDataTrack;
+export type AudioTrackPublication = LocalAudioTrackPublication | RemoteAudioTrackPublication;
+export type DataTrackPublication = LocalDataTrackPublication | RemoteAudioTrackPublication;
+export type VideoTrackPublication = LocalVideoTrackPublication | RemoteVideoTrackPublication;
+
+export interface EncodingParameters {
+  maxAudioBitrate?: number | null;
+  maxVideoBitrate?: number | null;
+}
+
+export type NetworkQualityLevel = number;
+
+export type NetworkQualityVerbosity = 0 | 1 | 2 | 3;
+
+export class NetworkQualityStats {
+  level: NetworkQualityLevel;
+  audio: NetworkQualityAudioStats | null;
+  video: NetworkQualityVideoStats | null;
+}
+
+export class NetworkQualityMediaStats {
+  send: NetworkQualityLevel;
+  recv: NetworkQualityLevel;
+  sendStats: NetworkQualitySendOrRecvStats | null;
+  recvStats: NetworkQualitySendOrRecvStats | null;
+}
+
+export class NetworkQualityAudioStats extends NetworkQualityMediaStats {}
+
+export class NetworkQualityVideoStats extends NetworkQualityMediaStats {}
```

Comment:
```suggestion
```

Reply:
@manjeshbhargav I can do this but I would need them for [NetworkQualityStats](https://media.twiliocdn.com/sdk/js/video/releases/2.8.0/docs/NetworkQualityStats.html). Should I still remove them? And if so, what should go in their place within `NetworkQualityStats`?